Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rails:4.2.6
+
+RUN echo "deb http://us.archive.ubuntu.com/ubuntu precise main universe" | tee -a /etc/apt/sources.list
+
+RUN mkdir /usr/local/node && curl -L https://nodejs.org/dist/v4.4.3/node-v4.4.3-linux-x64.tar.xz | tar Jx -C /usr/local/node --strip-components=1
+RUN ln -s ../node/bin/node /usr/local/bin/
+RUN ln -s ../node/bin/npm /usr/local/bin/
+
+# For running tests we need phantomjs
+RUN mkdir /usr/local/phantomjs && curl -L https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 | tar -xj -C /usr/local/phantomjs --strip-components=1
+RUN ln -s ../phantomjs/bin/phantomjs /usr/local/bin/
+
+WORKDIR /ohana-web-search
+
+COPY Gemfile /ohana-web-search
+COPY Gemfile.lock /ohana-web-search
+
+RUN gem install rubygems-update
+RUN update_rubygems
+RUN gem install bundler
```

Comment:
I'm curious why you're reinstalling bundler when it's already installed in the `rails` base image.